html-boilerplate/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<title>Tiffany's Story</title>
+	</head>
+	<body>
+		<section>
+			<h1>About</h1>
+			<p>I love to travel and help others book their travel. I was a mainframe developer for 15 years.</p>
+			<h2>Experience</h2>
+			<p> Total Systems Services, Alfa Life Insurance, Bullock County School System,
+			Math Tutor, T and J Travels</p>
+			<h3>Connect</h3>
+		<p> https://github.com/TiffanyD8/intro-to-programming-dorado
+			git@github.com:TiffanyD8/intro-to-programming-dorado.git</p>
```

Comment:
Let's just use the web site url here, since it's expected that someone would want to click a link here. Let's also wrap it an a <a> tag so it can be clicked on.
`<p><a href="https://github.com/TiffanyD8/intro-to-programming-dorado">My github page</a><p>`